src/openfermion/utils/operator_utils.py

```diff
@@ -184,6 +184,15 @@ def count_qubits(operator):
     elif isinstance(operator, PolynomialTensor):
         return operator.n_qubits
 
+    # Handle IsingOperator
+    elif isinstance(operator, IsingOperator):
+        num_qubits = 0
+        for term in operator.terms:
+            if term:
```

Comment:
Does this check for the term not being identity?  If so can you make it explicit?

Reply:
It doesn't, but any other case in this function doesn't do that either, so if we would be checking this only for Ising, that would be inconsistent.

I've checked and right now the tests for checking if `n_qubits` returns `5` are passing for all the following cases:

```
self.qubit_operator = QubitOperator("[] + [X1] + [Y2] + [Y3] + [Z4]")
self.qubit_operator = QubitOperator("[X0] + [X1] + [Y2] + [Y3] + [Z4]")
self.ising_operator = IsingOperator("[Z0] + [Z1] + [Z2] + [Z3] + [Z4]")
self.ising_operator = IsingOperator("[] + [Z1] + [Z2] + [Z3] + [Z4]")
```

Reply:
Huh...that seems broken..the first and last examples should return 4 no?

Reply:
I guess it depends how you want to count qubits.
If you want to act on the 4th qubit, you technically need 5 qubits, right?
We had some issues with how "empty" qubits are handled in various frameworks, @max-radin might have more context.

Reply:
Certainly in OpenFermion we don't minimize the qubit could.  If you generate a raising operator matrix on the fifth mode you will have a 2**5 matrix.  Cirq is more clever and will only work in the specified space of qubits.

Reply:
Makes sense!
Branch updated, thanks :)

Reply:
I did it in a non-cla compliant manner -.-
Will do it properly later.

Reply:
Ok, should be ok now!